downstream/modules/platform/proc-operator-external-db-controller.adoc

```diff
@@ -0,0 +1,55 @@
+
+[id="proc-operator-external-db-controller"]
+
+= Configuring an external database for {ControllerName} on {PlatformName} operator
+
+[role="_abstract"]
+For users who prefer to deploy {PlatformNameShort} with an external database, they can do so by configuring a secret with instance credentials and connection information, then applying it to their cluster using the `oc create` command.
+
+By default, the {PlatformName} operator automatically creates and configures a managed PostGreSQL pod in the same namespace as your {PlatformNameShort} deployment. A user may instead choose to use an external database if they prefer to use a dedicated node to ensure dedicated resources or to manually manage backups, upgrades, or performance tweaks. The following section outlines the steps to configure an external database for your {ControllerName} on a {PlatformNameShort} operator.
+
+.Prerequisite
+The external database must be a PostgreSQL database that is the version supported by the current release of {PlatformNameShort}.
+
+NOTE: {PlatformNameShort} 2.0 and 2.1 supports PostgreSQL 12
+
+.Procedure
+
+The external postgres instance credentials and connection information will need to be stored in a secret, which will then be set on the {ControllerName} spec.
+
+. Create a `postgres_configuration_secret` .yaml file, following the template below:
++
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: external-postgres-configuration
+  namespace: <target_namespace>
```

Comment:
It may make sense to make 2 more "help hints"  for `namespace` and `host` like you did for the other three, that way it is clear to the user that they need to modify those parts of the YAML provided.  

Potential wording.  
1. Namespace to create the secret in.  This should be the same namespace you wish to deploy to.  
2. The resolvable hostname of your database node.